tmt/cli.py

```diff
@@ -1074,3 +1076,94 @@ def lint(context, **kwargs):
         except SystemExit as e:
             exit_code |= e.code
     raise SystemExit(exit_code)
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+@main.group(cls=CustomGroup)
+@click.pass_context
+def setup(context, **kwargs):
+    """
+    Setup the environment for working with tmt.
+    """
+
+
+@setup.group(cls=CustomGroup)
+@click.pass_context
+def completion(context, **kwargs):
+    """
+    Setup shell completions.
+
+    By default, these commands only write a shell script to the output which
+    can then be sourced from the shell's configuration file.
```

Comment:
Perhaps we could mention the `--install` option here as well?

Reply:
Probably makes sense. Some new users may follow this "exploration flow":
```
tmt --help # I see setup command
tmt setup # I see completion command
tmt setup completion --help # I read that it only outputs the files
tmt setup completion bash
```
Some may not check the help of bash/zsh/fish completions since it sounds self-explanatory. Lets also mention it in the help